src/knot_cloud.c

```diff
@@ -421,11 +373,13 @@ int knot_cloud_register_device(const char *id, const char *name)
 	 * Expiration
 	 *	2000 ms
 	 */
-	result = mq_publish_direct_message(MQ_EXCHANGE_DEVICE,
-					   MQ_CMD_DEVICE_REGISTER,
-					   headers, 1,
-					   MQ_MSG_EXPIRATION_TIME_MS,
-					   json_str);
+	mq_message_data_t mq_message = {
+				 MQ_MESSAGE_TYPE_DIRECT, MQ_EXCHANGE_DEVICE,
+				 MQ_CMD_DEVICE_REGISTER, MQ_MSG_EXPIRATION_TIME_MS, json_str,
```

Comment:
We try to keep the code maximum column number at 80 characters (Sorry for still not having a contributor guide). Could you modify this part of code to fit in 80 characters?

Reply:
Strange. In my editor is 78 col length this line. it seems to me tab character is 8 spaces here in github